src/inspect.rs

```diff
@@ -14,11 +18,16 @@ pub fn inspect(opts: &InspectOptions) -> Result<()> {
         "cannot get vms for process {}",
         opts.pid
     );
+
     for map in vm.get_maps()? {
         println!(
             "vm mem: 0x{:x} -> 0x{:x} (physical: 0x{:x}, flags: {:?} | {:?})",
             map.start, map.end, map.phys_addr, map.prot_flags, map.map_flags,
         )
     }
+
+    let device = try_with!(Device::new(&Arc::new(vm)), "cannot create vm");
```

Comment:
I would maybe make a new subcommand called `attach` for this one, which will be eventually the command we use to attach to the VM.